Userland/Libraries/LibJS/JIT/Compiler.cpp

```diff
@@ -367,11 +367,11 @@ void Compiler::compile_increment(Bytecode::Op::Increment const&)
             Assembler::Operand::Register(GPR0),
             Assembler::Operand::Register(GPR1));
 
-        // if (GPR0 == 0x7fffffff) goto slow_case;
+        // if (GPR0 == INT32_MAX) goto slow_case;
         m_assembler.jump_if(
             Assembler::Operand::Register(GPR0),
             Assembler::Condition::EqualTo,
-            Assembler::Operand::Imm(0x7fffffff),
+            Assembler::Operand::Imm(INT32_MAX),
             slow_case);
```

Comment:
Please do this in a separate commit, as it has nothing to do with adding a decrement fast path.